security/totp-cli/Portfile

```diff
@@ -20,76 +20,65 @@ long_description    A simple TOTP (Time-based One-time Password) CLI tool. \
-                    github.com/kardianos/osext \
-                        lock    2bc1f35cddc0 \
-                        rmd160  9a45814aa98c9eb32b6c46ceb00e760d486c71ef \
-                        sha256  ed51f32d6729696e216d051b3955c22161a970aaef01c1819d85ad179e51ba41 \
-                        size    4904 \
-                    github.com/davecgh/go-spew \
+                        lock    v0.15.0 \
+                        rmd160  08e93131b3d22ceacee5f50d56f88ff2bb591df5 \
+                        sha256  bc663378acfc6d12da33eb2a58fccd7c8c3e254e86bcff44fe714088c05bff93 \
+                        size    1801178 \
+                    github.com/urfave/cli \
+                        lock    v2.25.7 \
+                        rmd160  54cb9b78d2762c931d2ec1f83015a8147158237e \
+                        sha256  a09c96d3722f1efdcba7677bc9ef9394947e22f7f1e235edf923607c3f8b942c \
+                        size    3482618 \
+                    github.com/cpuguy83/go-md2man \
+                        lock    v2.0.3 \
+                        rmd160  f44cb99228e4f418c00979bf850d568837755b76 \
+                        sha256  712375b6a4472b6eff9225cdf3e01a4d33e1e0753f713874ecd67a0d0c74bfea \
+                        size    64980 \
+                    github.com/xrash/smetrics \
+                        lock    039620a656736e6ad994090895784a7af15e0b80 \
+                        rmd160  55c9e9f554905046a0db05723db5a9d95c6b2d41 \
+                        sha256  996b007cfb8fd8308b8f1912bf3863a108edeb07e1e705b8294e13c7a3a662cb \
+                        size    1823438 \
+                    github.com/FiloSottile/age \
                         lock    v1.1.1 \
-                        rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
-                        sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
-                        size    42171
+                        rmd160  5e09dc3b85d53c92b62c114355097f51a8f79690 \
+                        sha256  007f2a349124a61c8357a8d34703f420248a4cd9c0c00892efab8186310baa8c \
+                        size    204319 \
+                    github.com/russross/blackfriday \
+                        lock    v2.1.0 \
+                        rmd160  c42a9332a2c2f3074c6f7e8d37a58d6148d2af08 \
+                        sha256  c4df56f2012a7d16471418245e78b5790569e27bbe8d72a860d7117a801a7fae \
+                        size    92950
+
+#
+# symlink for the "filippo.io/age" go package pointing to the "github.com/FiloSottile/age" go package
+# (since go-vendors cannot handle packages from filippo.io)
+#
+post-extract {
+                    file mkdir ${workpath}/gopath/src/filippo.io
+                    ln -s ${workpath}/gopath/src/github.com/FiloSottile/age ${workpath}/gopath/src/filippo.io/age
+}
 
 destroot {
-    xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
+                    xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
```

Comment:
Can we set the indentation for both the `destroot` and `post-extract` blocks to simply 4 spaces?